spec/lib/droplet_kit/resources/kubernetes_resource_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+RSpec.describe DropletKit::KubernetesResource do
+  subject(:resource) { described_class.new(connection: connection) }
+  include_context 'resources'
+
+  describe '#find' do
+    it 'returns a singular cluster' do
+      stub_do_api('/v2/kubernetes/clusters/20', :get).to_return(body: api_fixture('kubernetes/find'))
```

Comment:
perhaps change the fixture name to `kubernetes/find_cluster`. I think its helpful to specify because the node pools will be in here as well. unless you want to do `kubernetes/clusters/find`.

Reply:
good call out. i think i like the sub directory approach because then the file name matches the method name. i'll update.